Week3/js-exercises/ex1-AddSix.js

```diff
@@ -6,15 +6,23 @@ Declare a function called `createBase`.The function takes a number as a paramete
 return a closure, that adds a number to the base number argument.
 
 Call the function three times. The return values should be:
- 15, 24,  36
+ 15, 24,  33
 
  */
 
-function createBase( /* ???? */ ) {
+function createBase( number ) {
   // Put here your logic...
+  const plusNine = 9;
+   function addNumber() {
+     return (number += plusNine);
+  } 
+  return addNumber;
```

Comment:
correct but a bit too complicated
```suggestion
   return function () {
     number += 9;
     return number;
  } 
```